test/opt/common_uniform_elim_test.cpp

```diff
@@ -664,6 +664,307 @@ OpFunctionEnd
       predefs + before, predefs + after, true, true);
 }
 
+  TEST_F(CommonUniformElimTest, Volatile1) {
```

Comment:
Is this test any different that "Basic1"?  With no volatile values, what are you checking?

Reply:
Forgot to change the description: I changed one Load to Volatile.